domino/domino.py

```diff
@@ -73,7 +70,9 @@ def authenticate(self, api_key=None, auth_token=None, domino_token_file=None):
         """
         self.request_manager = _HttpRequestManager(get_auth_by_type(api_key=api_key,
                                                                     auth_token=auth_token,
-                                                                    domino_token_file=domino_token_file))
+                                                                    domino_token_file=domino_token_file,
+                                                                    api_key_from_env=os.getenv(DOMINO_USER_API_KEY_KEY_NAME),
+                                                                    domino_token_file_from_env=os.getenv(DOMINO_TOKEN_FILE_KEY_NAME)))
```

Comment:
I would revert this change, and handle all of the logic in `get_auth_by_type`. 
```suggestion
                                                                    domino_token_file=domino_token_file,
                                                                    api_key_from_env=os.getenv(DOMINO_USER_API_KEY_KEY_NAME))
```